docs/index.md

```diff
@@ -0,0 +1,32 @@
+# Jenkins-k8s Operator
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing [Jenkins](https://www.jenkins.io/) Agent on machines and configurable to use a Jenkins charm deployed in another Juju model.
+
+This charm simplifies initial deployment and "day N" operations of Jenkins Agent on VMs and bare metal.
+
+As such, the charm makes it easy for those looking to take control of their own agents whilst keeping operations simple, and gives them the freedom to deploy on the platform of their choice.
+
+For DevOps or SRE teams this charm will make operating Jenkins Agent simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing changes, and supports scaling out for enterprise deployments.
+
+## Project and community
+
+The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
```

Comment:
We should probably start linking https://app.element.io/#/room/#charmhub-charmdev:ubuntu.com instead

Reply:
got it, thanks for the tip. I fixed the link